index.html

```diff
@@ -276,6 +276,9 @@ <h1>Collection</h1>
           <span x-text="Math.floor(Object.keys(emojis).length / 100)  * 100"></span>
           emojis in total...
         </p>
+        <div class="hint-area" x-show="unlockedEmojis.length > 50" x-cloak>
```

Comment:
Let's go with a nicer round number
```suggestion
        <div class="hint-area" x-show="unlockedEmojis.length >= 100" x-cloak>
```